src/github.rs

```diff
@@ -5,8 +5,8 @@ use super::USERAGENT;
 use config::Config;
 
 pub fn release(config: &Config, tag_name: &str, tag_message: &str) -> Result<(), Error> {
-    let user      = &config.user[..];
-    let repo_name = &config.repository_name[..];
+    let user      = &config.user.as_ref().unwrap()[..];
+    let repo_name = &config.repository_name.as_ref().unwrap()[..];
```

Comment:
Ah, forget this remark here. We're slicing to get an actual `&str`, because it is necessary below for Hubcaps (which uses `Into<String>` for the method).
Keep the `&` and `[..]` above as is.